<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-R97L6J5" failures="1" tests="24" name="FirefoxTest" time="127.545" errors="0" timestamp="2019-10-28T12:50:04 IST">
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="checkLink" time="0.069"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="brokenLink" time="2.279"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="checktext" time="0.03"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="forgotPassWord" time="4.421"/>
  <testcase classname="com.filta.qa.login_test.LogIn_TestCase" name="signInTesting" time="19.152"/>
  <testcase classname="com.filta.qa.login_test.LogInInvalid_TestCase" name="signInWithInvalidCredentails" time="1.716"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="checkDropDown" time="0.096"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="forgotPassWordInvalid" time="2.002"/>
  <testcase classname="com.filta.qa.login_test.LogInInvalid_TestCase" name="signInWithInvalidCredentails2" time="1.675"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="footerCheck" time="0.033"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="forgotPassWordInvalid2" time="0.877"/>
  <testcase classname="com.filta.qa.login_test.LogInInvalid_TestCase" name="signInWithInvalidCredentails3" time="1.679"/>
  <testcase classname="com.filta.qa.login_test.LogInInvalid_TestCase" name="signInWithInvalidCredentails4" time="1.674"/>
  <testcase classname="com.filta.qa.login_test.LogInInvalid_TestCase" name="availableLinks" time="0.036"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="checkURL" time="0.074">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 1 out of bounds for length 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
at java.base/java.util.Objects.checkIndex(Objects.java:373)
at java.base/java.util.ArrayList.get(ArrayList.java:425)
at com.filta.qa.login_page.LoginPage.checkURL(LoginPage.java:111)
at com.filta.qa.login_test.LogInPage_TestCase.checkURL(LogInPage_TestCase.java:110)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- checkURL -->
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="forgotPassWordInvalid3" time="0.319"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="checkTextBox" time="0.025"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="forgotPassWordInvalid4" time="0.805"/>
  <testcase classname="com.filta.qa.login_test.LogInInvalid_TestCase" name="getBrokenLink" time="1.933"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="checkLabel" time="0.058"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="textboxCheck" time="0.023"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="availableLinks" time="0.048"/>
  <testcase classname="com.filta.qa.login_test.ForgetPassword_TestCase" name="labelCheck" time="0.293"/>
  <testcase classname="com.filta.qa.login_test.LogInPage_TestCase" name="getBrokenLink" time="1.33"/>
</testsuite> <!-- FirefoxTest -->
